tests/strategies/test_deepspeed_strategy.py

```diff
@@ -1185,3 +1185,24 @@ def test_deepspeed_multi_save_same_filepath(tmpdir):
     ckpt_path = os.path.join(trainer.checkpoint_callback.dirpath, "last.ckpt")
     expected = ["latest", "zero_to_fp32.py", "checkpoint"]
     assert set(expected) == set(os.listdir(ckpt_path))
+
+
+@RunIf(min_gpus=2, standalone=True, deepspeed=True)
+def test_deepspeed_with_bfloat16_precision(tmpdir):
```

Comment:
Just being mindful of precious CI time, could we skip the need to run a complete test? Maybe by ensuring if `precision=bf16` that the config object in the DeepSpeedStrategy contains the BF16 entry?

Reply:
we could, but we need to explicitly update the input types too to `bfloat`, so a quick test is good to have IMO. It runs with `fast_dev_run=True`.

Reply:
ah yes good point! sounds good :)